src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/ClientsApiServiceImpl.scala

```diff
@@ -84,4 +84,24 @@ final case class ClientsApiServiceImpl(authorizationProcessService: Authorizatio
       }
     }
   }
+
+  override def clientOperatorRelationshipBinding(clientId: String, relationshipId: String)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
+    toEntityMarshallerCreatedResource: ToEntityMarshaller[CreatedResource]
+  ): Route = {
+
+    val result: Future[CreatedResource] = for {
+      clientUuid       <- clientId.toFutureUUID
+      relationshipUuid <- relationshipId.toFutureUUID
+      result           <- authorizationProcessService.clientOperatorRelationshipBinding(clientUuid, relationshipUuid)
+    } yield (CreatedResource(result.id))
```

Comment:
which id is returned here, the client or the relationship?

Reply:
The client

Reply:
Is specializing the return type a good idea here maybe?

Reply:
Sure.